ajax/libs/cplayer/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "cPlayer",
```

Comment:
This field should be the same as folder name. In this case, we should use `"cplayer"`.